scripts/update_production.sh

```diff
@@ -40,9 +40,8 @@ touch disable-config-readonly
 
 ./vendor/bin/drush cache:rebuild --yes &&
 ./vendor/bin/drush updatedb --yes --cache-clear &&
-./vendor/bin/drush cs-update --discard-overrides --yes &&
+./vendor/bin/drush config:import --yes &&
 ./vendor/bin/drush search-api:reset-tracker --yes &&
-./vendor/bin/drush cache:rebuild --yes &&
```

Comment:
This no longer needed? Perhaps the cache is already cleared at the end of the `config:import`?

Reply:
We've added to many cache rebuilds because of our flawed update process. I'm confident that we don't need this anymore. Normally after a successful `config:import`, I'm expecting to have fresh data in cache. I'm proposing that we remove and keep an eye on next updates.